conf/test.routes

```diff
@@ -5,11 +5,11 @@ GET         /api/feature-switches                                   uk.gov.hmrc.
 + nocsrf
 POST        /api/feature-switches                                   uk.gov.hmrc.partnershipidentificationfrontend.featureswitch.api.controllers.FeatureSwitchApiController.updateFeatureSwitches()
 
-GET         /general-partnership/create-journey                     uk.gov.hmrc.partnershipidentificationfrontend.testonly.controllers.TestCreateGeneralPartnershipJourneyController.show
-POST        /general-partnership/create-journey                     uk.gov.hmrc.partnershipidentificationfrontend.testonly.controllers.TestCreateGeneralPartnershipJourneyController.submit
+GET         /general-partnership-journey                            uk.gov.hmrc.partnershipidentificationfrontend.testonly.controllers.TestCreateGeneralPartnershipJourneyController.show
+POST        /general-partnership-journey                            uk.gov.hmrc.partnershipidentificationfrontend.testonly.controllers.TestCreateGeneralPartnershipJourneyController.submit
 
-GET         /scottish-partnership/create-journey                    uk.gov.hmrc.partnershipidentificationfrontend.testonly.controllers.TestCreateScottishPartnershipJourneyController.show
-POST        /scottish-partnership/create-journey                    uk.gov.hmrc.partnershipidentificationfrontend.testonly.controllers.TestCreateScottishPartnershipJourneyController.submit
+GET         /scottish-partnership-journey                           uk.gov.hmrc.partnershipidentificationfrontend.testonly.controllers.TestCreateScottishPartnershipJourneyController.show
+POST        /scottish-partnership-journey                           uk.gov.hmrc.partnershipidentificationfrontend.testonly.controllers.TestCreateScottishPartnershipJourneyController.submit
```

Comment:
Could we change these to
/create-general-partnership-journey and
/create-scottish-partnership-journey , please? Just to match our other services

Reply:
done,  thanks